client/state/wp-data/calypso-wp-data-store.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { fetchPreferences, setPreference, savePreference } from 'state/preferences/actions';
+import { getPreference, isFetchingPreferences } from 'state/preferences/selectors';
+
+export default {
+	useCalypsoStore: true,
```

Comment:
Can we drop this bool flag, and just wrap the object in `use`, rather than [having the consumer do that](https://github.com/Automattic/wp-calypso/pull/26838/files?utf8=%E2%9C%93&diff=split#diff-0d47069349e64c0f7b035fefd21a44e9R6)? I think that would make a bit clearer what we're doing here.

Reply:
I'm not sure what you mean by this. Can you elaborate or share a code snippet?